scripts/test-fallback-scsv.py

```diff
@@ -46,6 +46,13 @@ def help_msg():
     print("                execution of preceding expected failure probe")
     print("                usage: [-x probe-name] [-X exception], order is compulsory!")
     print(" --tls-1.3      server does support TLS 1.3")
+    print(" --min-support val Minimum TLS version supported by server.")
+    print("                When set, the script will expect that CH messages")
```

Comment:
CH message - I presume client hello, I would consider not shortening.

Reply:
Good point, fixed